cypress/integration/buy-shirt.spec.ts

```diff
@@ -53,17 +56,30 @@ describe("Buy a t-shirt", () => {
     );
   });
 */
-  it.only("then the t-shirt should be bought POM", () => {
+  before(() => {
+    menuContentPage = new MenuContentPage();
+    plpPage = new PLPPage();
+    cartPage = new CartPage();
+    signinPage = new SigninPage();
+    addressesPage = new AddressesPage();
+    shippingPage = new ShippingPage();
+    paymentPage = new PaymentPage();
+    confirmPaymentPage = new ConfirmPaymentPage();
+    orderConfirmationPage = new OrderConfirmationPage();
+    userName = "tomas.alvarez@perficient.com";
+    password = "test123";
     menuContentPage.visitMenuContentPage();
+  });
+  it.only("then the t-shirt should be bought POM", () => {
     menuContentPage.goToTShirtMenu();
 
     plpPage.addToCartButton();
     plpPage.proceedToCheckoutButton();
 
     cartPage.proceedToCheckoutButton();
 
-    signinPage.emailAddressInput("tomas.alvarez@perficient.com");
-    signinPage.passwordInput("test123");
+    signinPage.emailAddressInput(userName);
+    signinPage.passwordInput(password);
     signinPage.signInButton();
```

Comment:
this can be just one method called signIn or login, and that method can receive the user and pass.